### ISSUE TITLE
6500kb limited, a strange phenomenon

### ISSUE THREAD

**Issue by xrandx** [tsudoko/anki-sync-server#15]
I upload my data normally until it reachs 6500kb. Whether it is anki2.0 or anki2.1.8, it always fails to upload when the data has already uploaded 6500kb. 
server: centos 7 python3.6

**Comment by AntonOfTheWoods** [comment]
I just today had a similar problem - the problem was not with ankisyncd but with the nginx proxy in front of it. I have just tested with a 12MB file after increasing the limit on nginx and now have no problem. Is your ankisyncd behind a proxy?

**Comment by xrandx** [comment]
> I just today had a similar problem - the problem was not with ankisyncd but with the nginx proxy in front of it. I have just tested with a 12MB file after increasing the limit on nginx and now have no problem. Is your ankisyncd behind a proxy?

No, I didn't use any proxy. The problem has been solved after I used the official server...

**Comment by AntonOfTheWoods** [comment]
@xrandx, You mean you synched with the official server and then pointed back to the custom server and the problem "fixed itself"?
